src/Essentials/samples/Samples/ViewModel/FlashlightViewModel.cs

```diff
@@ -30,6 +31,11 @@ public bool IsSupported
 			set => SetProperty(ref isSupported, value);
 		}
 
+		private async Task InitViewModel()
```

Comment:
drop private keyboard since it's the default

Reply:
I'll remove it, thanks!